cli/daemon/dash/dashapp/src/components/api/cm/CM.tsx

```diff
@@ -58,6 +58,10 @@ export default class CM extends React.Component<Props> {
     this.cm.on("focus", () => this.props.onFocus?.())
   }
 
+  componentWillUnmount() {
+    this.cm?.setValue("");
```

Comment:
Needed because of the new strict mode, will otherwise get two CodeMirror instances for each API request made